meilisearch/tests/index/test_index_search_meilisearch.py

```diff
@@ -60,9 +60,9 @@ def test_custom_search_params_with_wildcard(self):
             'a',
             {
                 'limit': 5,
-                'attributesToHighlight': '*',
-                'attributesToRetrieve': '*',
```

Comment:
The simple string `"*"` is not a valid entry ?

Reply:
It isn't for the post route. Do you think that's a core problem? A simple srting will creat a bad_request response